src/protocols/Protocols.h

```diff
@@ -74,12 +74,10 @@ class Id
     } // namespace name.
 
 CHIP_STANDARD_PROTOCOL(SecureChannel, 0x0000)             // Secure Channel Protocol
-CHIP_STANDARD_PROTOCOL(Echo, 0x0001)                      // Echo Protocol
+CHIP_STANDARD_PROTOCOL(InteractionModel, 0x0001)          // Interaction Model Protocol
 CHIP_STANDARD_PROTOCOL(BDX, 0x0002)                       // Bulk Data Exchange Protocol
 CHIP_STANDARD_PROTOCOL(UserDirectedCommissioning, 0x0003) // User Directed Commissioning Protocol
-CHIP_STANDARD_PROTOCOL(InteractionModel, 0x0005)          // Interaction Model Protocol
-CHIP_STANDARD_PROTOCOL(ServiceProvisioning, 0x0007)       // Service Provisioning Protocol
-CHIP_STANDARD_PROTOCOL(OpCredentials, 0x0008)             // Operational Credentials
+CHIP_STANDARD_PROTOCOL(Echo, 0x0004)                      // Echo Protocol.  To be removed or standardized.
 CHIP_STANDARD_PROTOCOL(TempZCL, 0x0009) // For carrying ZCL-compatible messages, to be removed when we no longer use them
```

Comment:
What still uses this? Could we remove this now that IM is fully supported?

Reply:
This is actually getting removed in #9842

Reply:
It's still used by some Android code and the code being removed in #9842